core/modules/widgets/text.js

```diff
@@ -31,7 +31,9 @@ TextNodeWidget.prototype.render = function(parent,nextSibling) {
 	this.computeAttributes();
 	this.execute();
 	var text = this.getAttribute("text",this.parseTreeNode.text || "");
+	var length = this.getAttribute("length");
```

Comment:
yes, but take care to add the second parameter, number "base" eg: 10 ... otherwise it will create strange results